components/treeview/accessibility/overview.md

```diff
@@ -0,0 +1,305 @@
+        public FontIcon? Icon { get; set; }
+        public bool Expanded { get; set; }
+
+        public TreeItem()
+        {
+
+        }
+
+        public TreeItem(TreeItem itmToClone)
+        {
+            this.Id = itmToClone.Id;
+            this.Text = itmToClone.Text;
+            this.ParentId = itmToClone.ParentId;
+            this.HasChildren = itmToClone.HasChildren;
+        }
+
+        public static TreeItem GetClonedInstance(TreeItem itmToClone)
+        {
+            return new TreeItem(itmToClone);
+        }
+    }
+
+    protected override void OnInitialized()
+    {
+        LoadFlatData();
+
+        ContextMenuData = new List<ContextMenuItem>()
+        {
+            new ContextMenuItem
+            {
+                Text = "Move before",
+                Icon = FontIcon.InsertTop,
+                CommandName = "MoveBefore"
+            },
+            new ContextMenuItem
+            {
+                Text = "Move after",
+                Icon = FontIcon.InsertBottom,
+                CommandName = "MoveAfter"
+            },
+            new ContextMenuItem
+            {
+                Text = "Add to parent",
+                Icon = FontIcon.InsertMiddle,
+                CommandName = "AddToParent"
+            },
+            new ContextMenuItem
+            {
+                Text = "Select",
+                Icon = FontIcon.CheckboxChecked,
+                CommandName = "ToggleSelect"
+            },
+            new ContextMenuItem
+            {
+                Separator = true
+            },
+            new ContextMenuItem
+            {
+                Text = "Delete",
+                Icon = FontIcon.Trash,
+                CommandName = "InvokeDelete"
+            }
+        };
+    }
+
+    private void LoadFlatData()
+    {
+        List<TreeItem>
+            items = new List<TreeItem>
+                ();
+
+        items.Add(new TreeItem()
+            {
+                Id = 1,
+                Text = "Project",
+                ParentId = null,
+                HasChildren = true,
+                Icon = FontIcon.Folder,
+                Expanded = true
+            });
+
+        items.Add(new TreeItem()
+            {
+                Id = 2,
+                Text = "Design",
+                ParentId = 1,
+                HasChildren = true,
+                Icon = FontIcon.Brush,
+                Expanded = true
+            });
+        items.Add(new TreeItem()
+            {
+                Id = 3,
+                Text = "Implementation",
+                ParentId = 1,
+                HasChildren = true,
+                Icon = FontIcon.Folder,
+                Expanded = true
+            });
+
+        items.Add(new TreeItem()
+            {
+                Id = 4,
+                Text = "site.psd",
+                ParentId = 2,
+                HasChildren = false,
+                Icon = FontIcon.FilePsd,
+                Expanded = true
```

Comment:
This looks like an outdated example, because the TreeView no longer uses `Expanded` model properties.